src/solvers/dgmulti/flux_differencing_gauss_sbp.jl

```diff
@@ -215,13 +215,13 @@ end
                                                      A::TensorProductGaussFaceOperator{2, Projection{ApplyFaceWeights}},
                                                      x::AbstractVector) where {ApplyFaceWeights}
 
-  @unpack interp_matrix_gauss_to_face_1d, face_indices_tensor_product = A
-  @unpack inv_volume_weights_1d, nnodes_1d, nfaces = A
+  (; interp_matrix_gauss_to_face_1d, face_indices_tensor_product) = A
+  (; inv_volume_weights_1d, nnodes_1d) = A
 
   fill!(out_vec, zero(eltype(out_vec)))
 
   # for 2D GaussSBP nodes, the indexing is first in x, then y
-  out = reshape(out_vec, nnodes_1d, nnodes_1d)
+  out = Base.ReshapedArray(out_vec, (nnodes_1d, nnodes_1d), ())
```

Comment:
Do you have a recipe when to use `reshape` and when to use `Base.ReshapedArray`?

Reply:
Would you mind adding some comments like these?

Reply:
Done